src/clojure_tictactoe/tictactoe.clj

```diff
@@ -17,9 +16,16 @@ Examples:
 ===+===+===  | ===+===+===  | ===+===+===  
    | x |     |  x | x | x   |    | x |     
 ===+===+===  | ===+===+===  | ===+===+===  
-   | x |     |    |   |     |    |   | x   ")
+   | x |     |    |   |     |    |   | x  \n\n ")
 
 
 (defn intro-game []
```

Comment:
I think this function name could be a bit more descriptive of what it's actually doing. Something like `print-game-introduction` might be a better alternative. What are your thoughts?